### DISCUSSION TITLE
How to use react.js with alpine.js?

### DISCUSSION THREAD

**Discussion by Thecode764** [alpinejs/alpine#4282] [1. Help] [answered]
Hi i use react.js, how to use alpine.js in react.js? 

__I use vite + react__

**Comment by Thecode764** [comment]
Notice: i use typescript

**Comment by ekwoka** [comment, marked as answer]
You don't.

There is no reason whatsoever to do that.

Both React and Alpine are UI frameworks for handling reactive data binding.

There is no reason you would ever use both together like that.

Why do you think you want to use Alpine in React?
